config/locales/de.yml

```diff
@@ -763,10 +771,12 @@ de:
     code: Code
     company: Firma
     complete: komplett
+    complete_order: Bestellung speichern
```

Comment:
I don't think this is the right term.

"Bestellung abschließen" would fit better here. Although technically we "save the order"  when you hit the button we should refer to what actually happens: we change the state to "complete"

Reply:
done